### ISSUE TITLE
AttributeError: 'Observation' object has no attribute 'observation'

### ISSUE THREAD

**Issue by deefrawley** [csparpa/pyowm#363]
Code taken directly from the online documentation:

```
from pyowm.owm import OWM

owm = OWM("XXX MY KEY XXX")

mgr = owm.weather_manager()

pressure_dict = mgr.weather_at_place('Berlin,DE').observation.pressure
pressure_dict['press']
pressure_dict['sea_level']
```
gives the following error

```
Traceback (most recent call last):
  File ".\owtest.py", line 7, in <module>
    pressure_dict = mgr.weather_at_place("Berlin,DE").observation.pressure
AttributeError: 'Observation' object has no attribute 'observation'
```

Happens for the rainfall example as well.
Someone has also posted this on [Stack Overflow](https://stackoverflow.com/questions/65376056/pyowm-observation-object-has-no-attribute-observation)

**Comment by csparpa** [comment]
@deefrawley thanks for pointing this out, it's a bug in the docs

The line `pressure_dict = mgr.weather_at_place('Berlin,DE').observation.pressure` 

should be

 `pressure_dict = mgr.weather_at_place('Berlin,DE').weather.pressure`

**Comment by csparpa** [comment]
Refer to: https://stackoverflow.com/questions/65376056/pyowm-observation-object-has-no-attribute-observation/65484048#65484048

**Comment by deefrawley** [comment]
No worries - I guess a documentation fix is much easier than code fix :)

**Comment by csparpa** [comment]
Fixed with 0eb016b29bda31f04284db3817d507de06f4a5f6
